### ISSUE TITLE
GOV.UK Footer component fails accessibility test

### ISSUE THREAD

**Issue by davet1985** [alphagov/govuk-frontend#5670]
## Description of the issue

It's been reported in an accessibility test that the "Support links" `h2` in the GOV.UK Footer component is causing a failure to comply. This failure is due to ordering of headings, meaning it only becomes an issue on screens which use headings down to the level 3 as it appears to a screenreader that the heading order is h1 -> h2 -> h3 -> h2, for example.

The suggestion is to remove the `h2` tag, but I'm hesitant because it's in this design system.

## Steps to reproduce the issue

Visit the DARTS login page: https://darts.staging.apps.hmcts.net/login and view the source for the footer.

This screenshot shows the `h2` Support links header.

![Screenshot 2024-10-17 at 08 57 53](https://github.com/user-attachments/assets/5b2ad7cd-c8b3-47ca-a953-e89fdc9b21f3)

## Actual vs expected behaviour

It's currently failing an accessibility test. I would expect using the GOV.UK Footer component not to result in a failure.

## Environment (where applicable)

- Operating system: MacOS Sonoma 14.5
- Browser: Chrome 
- Browser version: 129.0.6668.89 (Official Build) (arm64)

**Comment by selfthinker** [comment]
As beeps said, the heading levels as you described them are exactly how they are expected to be. That's neither a WCAG fail nor any other kind of accessibility or usability issue.

Regarding visually hidden headings confusing screen reader users, although it's possible to be confusing that something is read out when it isn't visible, it's also there to help screen reader users. Different screen reader users have different needs and different ways of navigating. While you can navigate by landmark (for example, jumping directly to the footer), navigating by heading is still the most common way to navigate with a screen reader. Someone who does that would need a heading at the start of the footer.

In the link you have provided the footer heading is now missing completely. It is not a good idea to remove the heading. If you don't like that heading is visually hidden, you can make it visible.
